cli/command/stack/common.go

```diff
@@ -0,0 +1,32 @@
+package stack
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+)
+
+// validateStackName checks if the provided string is a valid stack name (namespace).
+//
```

Comment:
nit: maybe remove this blank line?

Reply:
copy/paste :stuck_out_tongue_closed_eyes: